apps/example-orchestrator-ui/components/StartPage/NewProcessPanel.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactElement, useState } from 'react';
+import {
+    EuiButton,
+    EuiFieldSearch,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiPanel,
+    EuiSpacer,
+    EuiText,
+    EuiTextColor,
+} from '@elastic/eui';
+
+export default function NewProcessPanel(): ReactElement {
+    const [value, setValue] = useState('');
+    const onChange = (e) => {
+        setValue(e.target.value);
+    };
+    const frequentlyUsedValues = [
+        'IP Prefix',
+        'SN8 LightPath',
+        'NL8 MSC',
+        'NL8 Service Port 100G',
+        'SN8 AGGSP',
+    ];
+    const renderFrequentlyUsed = (values) => {
+        return (
+            <EuiFlexGroup gutterSize="m">
+                {values.map((value, index) => (
+                    <EuiFlexItem key={index} grow={false}>
+                        {/*@ts-ignore*/}
```

Comment:
Try to avoid ts-ignores, and when you add a TS ignore explain wit a commentblock in code why

Reply:
"xs" size is not assignable for the EuiButton ("s" is minimum), however if we pass "xs" it works completely fine and we can make it same as the prototype